spec/services/comment_maker_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+describe CommentMaker do
+
+  let(:notifier_mock) do
+    double(:comment_notifier, call: nil)
+  end
+
+  let(:params) do
+    {
+        comment_text: "No agree",
+        debate_id: 123
+    }
+  end
+
+  let(:user) do
+    create(:slack_user)
+  end
+
+  subject do
+    CommentMaker.new(notifier_mock)
+  end
+
+  describe "#call" do
+    it "creates a new comment" do
+      expect {
+        subject.call(params)
+      }.to change(Comment, :count).by(1)
+    end
+
+    it "assigns a comment to a correct user if given" do
```

Comment:
can you add a test checking that anonymous_user will be returned when slack user is missing ?